content/docs/actors/development/state_persistence.md

```diff
@@ -43,8 +43,10 @@ The [Apify SDK](https://sdk.apify.com) persists its state automatically, using t
 To persist state manually, you can use the [Apify.events](https://docs.apify.com/actors/development/state-persistence) method in the Apify SDK.
 
 ```javascript
-Apify.events.on('migrating', () => {
-    Apify.setValue('my-crawling-state', {
+import { Actor } from 'apify';
+
+Actor.on('migrating', () => {
+    Actor.setValue('my-crawling-state', {
         foo: 'bar',
     });
 });
```

Comment:
Let's not worry about this now, but maybe we could add `useState` to the SDK as well? https://crawlee.dev/docs/upgrading/upgrading-to-v3#auto-saved-crawler-state

Reply:
All what useState does is already available, this is the implementation:

```ts
async useState<State extends Dictionary = Dictionary>(defaultValue = {} as State): Promise<State> {
    const kvs = await KeyValueStore.open(null, { config: this.config });
    return kvs.getAutoSavedValue<State>(BasicCrawler.CRAWLEE_STATE_KEY, defaultValue);
}
```

So we can just expose `Actor.getAutoSavedValue` as a static helper with some default state key, or even have an `Actor.useState` for that, if it would use some static state key that might be more in line.